velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -110,12 +110,13 @@ ::duckdb::Value duckValueAt<TypeKind::ARRAY>(
     return ::duckdb::Value::EMPTYLIST(duckdb::fromVeloxType(elements->type()));
   }
 
-  std::vector<::duckdb::Value> array;
+  ::duckdb::vector<::duckdb::Value> array;
   array.reserve(size);
   for (auto i = 0; i < size; i++) {
     auto innerRow = offset + i;
     if (elements->isNullAt(innerRow)) {
-      array.emplace_back(::duckdb::Value(nullptr));
```

Comment:
Looks like this change is responsible for breaking Join Fuzzer: https://github.com/facebookincubator/velox/issues/7943

CC: @majetideepak @kgpai @pedroerp